src/main/java/org/tranqwave/fobwebapilambda/section/GetSectionLambda.java

```diff
@@ -0,0 +1,32 @@
+package org.tranqwave.fobwebapilambda.section;
+
+import dao.SectionDao;
+import dao.dbModels.DynamoDBSection;
+import model.GetSectionResponse;
+import model.Request;
+import model.Section;
+
+import javax.naming.Context;
+
+public class GetSectionLambda {
+    private final SectionDao sectionDao;
+
+    public GetSectionLambda(SectionDao sectionDao) {this.sectionDao = sectionDao;}
+
+    public GetSectionResponse getSection(Request request) {
+        final DynamoDBSection dynamoDBSection =  sectionDao.getSection(request.toString());
```

Comment:
If we were to implement the GetSection API you would need a GetSection request for this lambda, that would contain the specific section_id were looking for.